README.md

```diff
@@ -115,6 +115,20 @@ To create a new theme definition:
  - Create a file *[NAME-OF-THE-THEME_DEFINITION].css* and/or *[NAME-OF-THE-THEME_DEFINITION].js* to get these files
   automatically loaded.
   
+Atomium provides a custom page available on the path: **atomium-overview**.
+This particular page is only available to users with _administer themes_ permission.
+
+This page act as a showcase page of components.
+To add a component in there, your component needs to define one or two hooks:
+ - hook_atomium_definition_hook().
+   
+   This hook allows you to define simply a component.
```

Comment:
We can add here the option 
'disable' => TRUE, ?

Reply:
I don't think it's necessary as I describe this in the .api.php file.

Reply:
Ok, it's true.